{{cookiecutter.project_slug}}/backend/app/app/tests/utils/user.py

```diff
@@ -10,41 +10,43 @@
 from app.tests.utils.utils import random_email, random_lower_string
 
 
-def user_authentication_headers(
+async def user_authentication_headers(
     *, client: TestClient, email: str, password: str
```

Comment:
Should update all these `TestClient` annotations to `AsyncClient`

Reply:
Updated, thanks!